README.md

```diff
@@ -84,4 +84,30 @@ If the default `SymbolProvider` is missing useful information for the language /
 
 ## `AutocompleteManager` API
 
-The `AutocompleteManager` is exposed as a [provided service](http://flight-manual.atom.io/behind-atom/sections/interacting-with-other-packages-via-services/). Its main public method is `watchEditor` which allows external editors to register for autocompletions from providers with a given set of labels.
+The `AutocompleteManager` is exposed as a [provided service](http://flight-manual.atom.io/behind-atom/sections/interacting-with-other-packages-via-services/).
+The service, named `autocomplete.manager`, provides a function that can execute a callback on the `AutocompleteManager` object.
+External packages can access this service with the following code.
+
+In `package.json`:
+```
+{
+  "consumedServices": {
+    "autocomplete.manager": {
+      "versions": {
+        "1.0.0": "consumeAutocompleteManager",
+      }
+    }
+  }
+}
+```
+In the main module file:
+```
+consumeAutocompleteManager(service) {
```

Comment:
What about the same thing that returns a promise resolving to a disposable, I imagine it working like this...

```js
// In a package that consumes the service

async consumeWatchEditorService (watchEditor) {
  const disposable = await watchEditor(editor, labels)
  // ...
}
```

One thing I'm now wondering is why does the autocomplete manager need to be required asynchronously. Is it a performance thing? Because just having a simple synchronous API seems a lot more sane.